### PR TITLE
BigQuery SQL query step

### PR DESCRIPTION
**Pull request by JuhaKiili**
Basic SQL query step for BigQuery.

Supports IAM or keyfile contents passed as an environment variable.

### CODE REVIEW

File: connectors/steps/bigquery/query.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ) as conn:
+        print(sql_query + "\n")
+
+        rows = ()
+        columns = []
+
+        query_job = conn.query(sql_query)
+        result = query_job.result()
+        columns = [field.name for field in result.schema]
+        rows = tuple([str(cell) for cell in row] for row in result)
```
Comment (akx) [inline connectors/steps/bigquery/query.py:36]: Why is this necessary? Doesn't the CSV serializer deal with stringifying columns?

Comment (akx) [reply connectors/steps/bigquery/query.py:36]: Ah, it looks like `CSVSerializer`'s `serialize`'s signature mismatches with the parent class – this really isn't necessary, mypy just makes you think it is...

**Comment by JuhaKiili** [comment]
@akx  I'm unsure if this PR is now blocked until #8 goes in?

What are my next steps for getting this PR in?

**Review by akx — APPROVED** [review]
